Models/User.cs

```diff
@@ -0,0 +1,7 @@
+﻿using Microsoft.AspNetCore.Identity;
+
+namespace API.Models
+{
+    public class User : IdentityUser {
+    }
```

Comment:
Maybe we should format this as `public class User : IdentityUser { }` because it is highly likely this class will stay like this for the duration of the project.